app/code/Magento/SendFriend/view/frontend/templates/send.phtml

```diff
@@ -123,7 +123,7 @@
 <script type="text/x-magento-init">
     {
         "a[role='back']": {
-            "Magento_SendFriend/back-event": {}
+            "Magento_SendFriend/js/back-event": {}
```

Comment:
We need to create mapping between old and new path in requirejs-config.js file in Magento_SendFriend module

Reply:
Applied changes as per our slack conversion.